test/test_build.py

```diff
@@ -359,14 +355,37 @@ def test_image_boots(image_type):
         # XXX: read the fully yaml instead?
         assert f"image: {image_type.container_ref}" in output
 
-        # Figure out how big / is and make sure it is > 11bGiB
-        # Note that df output is in 1k blocks, not bytes
-        for line in output.splitlines():
+        # check the minsize specified in the build configuration for each mountpoint against the sizes in the image
+        exit_status, output = test_vm.run("df --output=target,size", user="root",
+                                          keyfile=image_type.ssh_keyfile_private_path)
+        assert exit_status == 0
+        # parse the output of 'df' to a mountpoint -> size dict for convenience
+        mountpoint_sizes = {}
+        for line in output.splitlines()[1:]:
             fields = line.split()
-            if fields[0] == "/sysroot":
-                size = int(fields[1])
-                assert size > 11 * 1024 * 1024
-                break
+            # Note that df output is in 1k blocks, not bytes
+            mountpoint_sizes[fields[0]] = int(fields[1]) * 2 ** 10  # in bytes
+
+        fs_customizations = testutil.create_filesystem_customizations(image_type.rootfs)
```

Comment:
(nitpick) maybe worth extracting this into a small helper like: `assert_fs_customizations(image_type, mountpoint_sizes) ` to keep the `test_image_boots` function slightly smaller?

Reply:
Done!